modules/BookCollection.js

```diff
@@ -0,0 +1,45 @@
+import Book from './Book.js';
+
+export default class BookCollection {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  // Using classes instead of objects for add book function.
+  addBook = (title, author) => {
+    const book = new Book(title, author);
+
+    this.books.push(book);
+    this.saveToLocalStorage();
+  }
+
+  removeBook = (title) => {
+    this.books = this.books.filter((book) => book.title !== title);
+    this.saveToLocalStorage();
+  }
+
+  displayBooks() {
+    const bookList = document.getElementById('book-lists');
+    bookList.innerHTML = '';
+
+    this.books.forEach((book) => {
+      const listItem = document.createElement('li');
+      listItem.style.listStyle = 'none';
+      listItem.classList.add('list-item');
+
+      const rmvBtn = document.createElement('button');
+      rmvBtn.textContent = 'Remove';
+      rmvBtn.classList.add('remove-button');
+      rmvBtn.dataset.title = book.title;
+
+      listItem.innerHTML = `"${book.title}" by ${book.author}`;
+
+      bookList.appendChild(listItem);
+      bookList.appendChild(rmvBtn);
+    });
+  }
```

Comment:
- [ ] 👍 As I mentioned in my previous review, Some of your function does not follow the ES6 syntax. I would recommend utilizing arrow functions instead of regular functions. This practice aligns with the ES6 standard and enhances the readability and maintainability of the code. Please consider updating your code to use arrow functions wherever appropriate. Keep up the good work, and happy coding! 🚀🎉
- [ ] **Check all functions in all files and use arrow functions instead of regular functions👍**